.github/workflows/e2e.yaml

```diff
@@ -60,6 +60,10 @@ jobs:
         with: 
           go-version-file: 'go.mod'
           cache: false
+      - name: Install Kubectl
+        run: |
+           curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
is there some reason we're not using `k3s kubectl`? If we have the k3s binary available, we should be able to use its embedded kubectl, even if we don't run the install script.

Reply:
It comes down to how we use the `vagrant ssh -c`. This solves problems around the double "double quotes" issue. Using the `RunCmdOnNode("kubectl", "server-1")` results in a command on the host of `vagrant ssh server-1 -c "kubectl"`, which causes issues around too many quotes. Instead, we install kubectl on the host, produce a kubeconfig from the server-0 node and then access kubectl on the host for the `RunCommand("kubectl")` calls.